### PR TITLE
PPTP-327: check your answers page for liability details

### PR DESCRIPTION
**Pull request by tombart**

### CODE REVIEW

**Review by PauloPortugal — CHANGES_REQUESTED** [review]
Looks awesome! 
It would be good to reuse some of the test code, but happy to discuss it.

**Comment by PauloPortugal** [comment]
It would be good to review the other controllers and reuse the code as well, this way w avoid having more code to maintain on each controller.

File: conf/messages.en [hunk 1]
```diff
@@ -10,6 +10,8 @@ site.back = Back
 site.button.continue = Save and Continue
 site.error.summary.title = There is a problem
 
+answers.page.change.link = Change
```
[the remark this answers is not in the document]
Comment (tombart) [reply conf/messages.en:13]: done

File: test/base/unit/ControllerSpec.scala [hunk 2]
```diff
@@ -52,4 +54,12 @@ trait ControllerSpec
   def getRequest(): Request[AnyContentAsEmpty.type] =
     FakeRequest("GET", "").withCSRFToken
 
+  def givenRegistrationExistOnRequest() = {
+    val registration = aRegistration()
+    given(mockRegistrationConnector.find(any())(any())).willReturn(
+      Future(Right(Some(registration)))
+    )
+    registration
+  }
```
[the remark this answers is not in the document]
Comment (tombart) [reply test/base/unit/ControllerSpec.scala:63]: done

[the remark this answers is not in the document]
Comment (tombart) [reply test/base/unit/ControllerSpec.scala:63]: done

File: test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/CheckLiabilityDetailsAnswersControllerTest.scala [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+        intercept[RuntimeException](status(result))
+      }
+    }
+
+    "redirects to registration page" when {
+      "user proceed" in {
+        authorizedUser()
+
+        val result = controller.submit()(FakeRequest("POST", ""))
```
[the remark this answers is not in the document]
Comment (tombart) [reply test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/CheckLiabilityDetailsAnswersControllerTest.scala:80]: done

File: test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/CheckLiabilityDetailsAnswersControllerTest.scala [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.controllers
+
+import akka.http.scaladsl.model.StatusCodes.OK
```
[the remark this answers is not in the document]
Comment (tombart) [reply test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/CheckLiabilityDetailsAnswersControllerTest.scala:19]: done

File: test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/CheckLiabilityDetailsAnswersControllerTest.scala [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+  "Check liability details answers Controller" should {
+
+    "return 200" when {
+
+      "user is authorised and display page method is invoked" in {
+        authorizedUser()
+        val result = controller.displayPage()(fakeRequest)
+
+        status(result) mustBe OK.intValue
```
[the remark this answers is not in the document]
Comment (tombart) [reply test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/CheckLiabilityDetailsAnswersControllerTest.scala:62]: removed

**Comment by tombart** [comment]
yeap, valid point, didn't look properly what's there already, I'll review those as a part of this pr

**Review by PauloPortugal — APPROVED** [review]

**Pull request merged**